src/BigQuery/Table.php

```diff
@@ -86,6 +86,21 @@ public function exists()
     }
 
     /**
+     * Delete the table.
+     *
+     * Example:
+     * ```
+     * $table->delete();
+     * ```
+     *
+     * @see https://cloud.google.com/bigquery/docs/reference/v2/tables/delete Tables delete API documentation.
+     */
+    public function delete()
```

Comment:
Can we please add `$options` as parameter here so it is possible to configure request options.